src/styles/ProjectSummary.scss

```diff
@@ -0,0 +1,12 @@
+.ProjectSummary{
+    .MuiBox-root-14{
```

Comment:
Should there be space before the opening brace? @sofiarm21 not sure if we are being strict about this..